lib/astronoby/angle.rb

```diff
@@ -47,24 +47,30 @@ def as_dms(degree, minute, second)
       end
     end
 
+    attr_reader :value
+
+    def initialize(value)
+      @value = if value.is_a?(Integer) || value.is_a?(BigDecimal)
```

Comment:
Thoughts on naming the instance variable `@radians` for readability? That way there's no ambiguity internally that value is stored in radians.

Reply:
That's a great idea 👍